src/components/index.d.ts

```diff
@@ -373,40 +373,40 @@ export declare const EmailInput: {
 export declare const TextMaskedInput: {
   (props: any): JSX.Element
   PUBLIC_PROPS: {
-    doValidation: any
-    placeholder: any
     mask: any
-    guide: any
-    initialValue: any
-    keepCharPositions: any
-    pipe: any
-    type: any
-    'data-tid': any
-    disabled: any
-    allCaps: any
-    name: any
-    labelCopy: any
-    validator: any
-    setTouched: any
-    getTouched: any
+    type: string
+    'data-tid': string
+    name: string
+    labelCopy: string
+    doValidation?: any
+    placeholder?: any
+    guide?: boolean
+    initialValue?: string
+    keepCharPositions?: boolean
+    pipe?: any
+    disabled?: boolean
+    allCaps?: boolean
+    validator?: any
+    setTouched?: any
+    getTouched?: boolean
   }
   propTypes: {
-    doValidation: any
-    placeholder: any
     mask: any
-    guide: any
-    initialValue: any
-    keepCharPositions: any
-    pipe: any
-    type: any
-    'data-tid': any
-    disabled: any
-    allCaps: any
-    name: any
-    labelCopy: any
-    validator: any
-    setTouched: any
-    getTouched: any
+    type: string
+    'data-tid': string
+    name: string
+    labelCopy: string
+    doValidation?: any
```

Comment:
If this is a function, the type signature should be something like this:

```ts
  pipe?: () => void // or any other return type: any, T (Generic), boolean, etc)
```

Reply:
Thanks @bwalker8038 -- I was going to save the functions for another pass, but I think I'll just go ahead and start on a follow up now!